src/ontology/mondo-ingest.Makefile

```diff
@@ -157,13 +157,13 @@ metadata/mondo.sssom.config.yml:
 mappings: sssom $(ALL_MAPPINGS)
 
 #################
-# Utils #########
+##### Utils #####
 #################
 # Documentation for `report-mapping-annotations` and `update-jinja-sparql-queries`: `docs/developer/ordo.md`
 # TODO: When https://github.com/monarch-initiative/mondo-ingest/issues/43 is fixed, can change back to `requirements.txt`
 python-install-dependencies:
 	python3 -m pip install --upgrade pip
-	python3 -m pip install -r $(RELEASEDIR)/requirements-unlocked.txt
+	python3 -m pip install --upgrade -r $(RELEASEDIR)/requirements-unlocked.txt
```

Comment:
@matentzn This is necessary for OAK I think. But we may want to restrict this `--upgrade` to just a few cases. See: #57

Reply:
sounds good, I think this is needed for sssom as well..